scripts/webidl/convert.js

```diff
@@ -46,4 +46,4 @@ function onlyIDL(filePath) {
   return path.extname(filePath) === ".idl";
 }
 
-doConversion("lib/jsdom/living/generated/events").done();
+doConversion("lib/jsdom/living/events").done();
```

Comment:
But then we'd have all files in a single location, that'll get confusing really quickly I'd imagine.

Or do you mean the convert script should group according to folders by itself?